src/test/java/hudson/plugins/build_timeout/impl/ElasticTimeOutStrategyTest.java

```diff
@@ -40,15 +39,15 @@ public void percentageWithTwoBuilds() throws Exception {
 
         Build b = new Build(new Build(20 * MINUTES, SUCCESS, new Build(40 * MINUTES, SUCCESS)));
 
-        assertEquals("Timeout should be 200% of the average of 20 and 40", 60 * MINUTES, strategy.getTimeOut(b,null));
+        assertEquals( 60 * MINUTES, strategy.getTimeOut(b,null),"Timeout should be 200% of the average of 20 and 40");
```

Comment:
Would it be possible to remove the space before the `60 * MINUTES` here?

Reply:
Done